tests/async/test_valgrind_shutdown_cleanup.py

```
@@ -74,3 +75,70 @@ def body(source, start, end):
+    assert(message.empty());
+    assert(would_autoreboot());
+    shutdownData.reboot_time = 0;
+    assert(would_autoreboot());
+}
+'''
+build = root / "bin/tests/shutdown-status"
+build.mkdir(parents=True, exist_ok=True)
+source = build / "regression.cpp"
+binary = build / "regression"
+source.write_text(harness)
+subprocess.run(["g++", "-std=c++20", str(source), "-o", str(binary)], check=True)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)